.github/workflows/check-pr.yml

```diff
@@ -0,0 +1,38 @@
+name: check-pr
+on:
+  pull_request:
+    types: [opened, reopened, labeled, unlabeled, synchronize]
+  pull_request_review:
+    types: [submitted, dismissed]
+jobs:
+  check-labels:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: RIOT-OS/check-labels-action@v1.0.0
+      with:
+        access_token: ${{ secrets.GITHUB_TOKEN }}
+        unset_labels: 'CI: needs squashing,State: waiting for other PR'
+        cond_labels: '(Process: needs >1 ACK,review.approvals>1),(Area: RDM,review.approvals>2)'
+  check-commits:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+      with:
+        # Use the SHA of the PR branch as-is, not the PR branch merged
+        # in master (default behavior in GH actions)
+        # See https://github.com/actions/checkout#checkout-pull-request-head-commit-instead-of-merge-commit
+        ref: ${{ github.event.pull_request.head.sha }}
+        fetch-depth: 0
+    - name: Fetch base branch
+      run:
+        git fetch origin '${{ github.base_ref }}:${{ github.base_ref }}'
+    - name: Run checks
+      env:    # enforce deactivation ubuntu label check
+        TRAVIS_PULL_REQUEST:
```

Comment:
This variable could be renamed to better reflect that this is running on GH actions, or maybe rename it to something more generic (`CI_PULL_REQUEST` ?)

Reply:
The variable is defined within `check_pr`. This `env` is mostly to ensure that it isn't set by some spurious environment ;-)'

Reply:
Yes I understand that. pr_check.sh can be updated of course.

Reply:
There is some clean-up required in that script in any case. Should we put that to a follow-up maybe and stick to the current workaround for now? I updated the comment to be a bit clearer on that.

Reply:
Done that cleanup.